pyabc2/pitch.py

```diff
@@ -498,6 +498,28 @@ def _repr_html_(self):
         cn = self.to_pitch_class()._repr_html_()
         return f"{cn}<sub>{self.octave}</sub>"
 
+    def helmholtz(self) -> str:
+        """Note name in Helmholz pitch notation, e.g. "C," and "c'"."""
```

Comment:
```suggestion
        """Pitch name in Helmholtz pitch notation, e.g. ``C,`` and ``c'`` (ASCII)."""
```
I know it isn't very consistent currently, but I try to distinguish between note (pitch + duration) and pitch (no duration), since they each have their own class.